src/components/RepositoryNotifications.tsx

```diff
@@ -55,15 +57,20 @@ export const RepositoryNotifications: FC<IRepositoryNotifications> = ({
         className="group flex items-center justify-between bg-gray-100 px-3 py-2 dark:bg-gray-darker dark:text-white"
         onClick={toggleRepositoryNotifications}
       >
-        <div className="mt-0 flex flex-1 space-x-3 overflow-hidden overflow-ellipsis whitespace-nowrap text-sm font-medium">
+        <div
+          className={cn(
+            'flex flex-1 space-x-3 overflow-hidden overflow-ellipsis whitespace-nowrap text-sm font-medium',
+            Opacity.MEDIUM,
```

Comment:
Does the change in opacity look okay in the icon? But I like the consistency with the other change in NotifRow

Reply:
I think it does.  Applying it at the row level seemed to make the opacity hierarchy clearer